pkg/client/unversioned/horizontalpodautoscaler_test.go

```diff
@@ -74,6 +75,7 @@ func TestHorizontalPodAutoscalerGet(t *testing.T) {
 	}
 
 	response, err := c.Setup(t).Extensions().HorizontalPodAutoscalers(ns).Get("abc")
+	defer c.server.Close()
```

Comment:
Can you make a `Close()` method on the `testClient` instead of knowing which field `Setup` initializes?